lib/services/ios-provision-service.ts

```diff
@@ -90,6 +103,31 @@ export class IOSProvisionService {
 
 		return { devices, match };
 	}
+
+	public async getDevelopmentTeams(): Promise<{ id: string, name: string }[]> {
```

Comment:
This is used once, if I have to write it out in 2-3 places perhaps I will make it a separate interface. For now type inference works so well, this type is never written out anywhere else in the code.